.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,15 @@
 - [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
 - [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.
 
+#### For EI only PR
+* When creating PR:
+- [ ] I've modified ```src/config/build_config.py``` in my PR branch by disabling all non-EI purpose frameworks and enabling related EI purpose frameworks
+* When PR is reviewed and ready to be merged:
+- [ ] I've revert the code change on the config file mentioned above
+
+* Before merging, for PR merger, please cross-check:
```

Comment:
PR merger does not sound right,

Reply:
Sorry I am also considering for a better way. Which word should I use?